plugins/com.aptana.js.debug.core/src/com/aptana/js/debug/core/internal/model/JSDebugTarget.java

```diff
@@ -2180,39 +2175,42 @@ protected URI resolveSourceFile(String sourceFile)
 		try
 		{
 			URI uri = new URI(sourceFile);
-			String scheme = uri.getScheme();
-			if (FILE.equals(scheme))
+			if (!uri.isOpaque())
```

Comment:
Adding a guard might help to avoid NPE, however, they would still won't be able to debug the app. The underlying issue seems to be with getting the invalid path from the SDK debugger.

I realized that we log the debug protocol communication at  ``<workspace>/.metadata/.plugins/com.appcelerator.titanium.ios.core/logs/iosdebugger.log`` and asked the customer for these logs.